modules/machineset-modifying.adoc

```diff
@@ -8,9 +8,15 @@
 [id="machineset-modifying_{context}"]
 = Modifying a compute machine set by using the CLI
 
+You can modify the configuration of a compute machine set and then propagate the changes to the machines in your cluster by using the CLI.
+
+By updating the compute machine set configuration, you can enable features or change the properties of the machines it creates.
+You might also need to update the compute machine set so that a change made in the underlying cloud is reflected in the compute machine sets in your cluster and propagated to the machines they create.
```

Comment:
I'd shift this sentence after the next two sentences and rephrase a bit to be more ambiguous about support for direct-cloud-side-changes here, and to instead stick to how the components .  What about something like:

> By updating the compute machine set configuration, you can enable features or change the properties of the machines it creates.
> When you modify a compute machine set, your changes only apply to compute machines that are created after you save the updated `MachineSet` custom resource (CR).
> The changes do not affect existing machines.
> Changes made in the underlying cloud provider are not reflected in the Machine or MachineSet resources; to adjust instance configuration in cluster-managed infrastructure, use the cluster-side resources.